Modules/xASL_module_ASL.m

```diff
@@ -187,6 +188,22 @@
 	x.Q.NumberEchoTimes = 1;
 end
 
+%% D6. DeltaM parsing - check if all/some volumes are deltams
+% If TSV file exist
+[pathDir, pathASL4D] = fileparts(x.P.Path_ASL4D);
+pathASL4Dcontext = fullfile(pathDir, [pathASL4D '_aslcontext.tsv']);
```

Comment:
perhaps good to use the same file naming as the rest, so this would be `x.P.Path_ASL4Dcontext`. Even better if you can add this in the general file used for creating all paths (`xASL_ini_FileSystem`), so you don't have to repeat this in different places

Reply:
Good point. Will do.

Reply:
I've checked this function and it needs a revamp. It creates a HUGE structure of 30.000 cells. 
I've made a new issue for that #1006 and I will add the ASL4Dcontext path there. It makes no sense to add this new path there quickly without revamping the function as it would only make it grow even more.